test/manual/api/system_api_tests.sh

```
@@ -307,6 +305,17 @@ create_connections() {
 
   echo "[SCRIPT] Batch upsert response: $response"
   
+  # Debug: List all connections after batch upsert
+  echo "[SCRIPT] Listing all connections after batch upsert:"
+  local list_raw=$(make_request GET "$API_BASE_URL/api/maps/$MAP_SLUG/systems")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)